snuba/query/expressions.py

```diff
@@ -132,6 +132,10 @@ class Column(Expression):
 
     table_name: Optional[str]
     column_name: str
+    # For nested column, this is the path following the main column name.
+    # For tags.key as an example, tags is the column_name and (value,) would
+    # be the path
+    path: Tuple[str, ...] = tuple()
```

Comment:
From the previous version of this change that was in #963, there was a question whether it would be worth to support multiple levels even if clickhosue does not.
I don't have a strong opinion on this, so I am open in turning it to a string.
Only point to keep in mind is that this is still shared between the logical schema and physical schema so it is not only clickhouse specific.